timepiece/entries/tests/test_timesheet.py

```diff
@@ -1318,7 +1318,7 @@ def test_user_edit_approved_entry(self):
         self.assertEqual(response.status_code, 404)
 
     def test_edit_invoiced_entry(self):
-        """You shouldnt be able to edit an invoiced entry"""
```

Comment:
I would prefer to see the superuser test case added, and retain the test that proves a 'normal' user cannot edit in this case.